### ISSUE TITLE
Website hijacked after installing Joomla 3.4.8

### ISSUE THREAD

**Issue by ericplugge** [joomla/joomla-websites#202]
This morning I've updated my Joomla site from 3.4.4 to 3.4.8 thru Administrator. A few moments later the website' files have the extension .ENCRYPTED and indeed are not readilble anymore. I also found a README_FOR_DECRYPT.txt file with payment instructions.

The website is hosted at www.mijndomein.nl, and well known provider in the Netherlands.

Of course I don't know how this was possible. But it looks like the contamination came with the update. So, I guess you should warn others too, and maybe prevent the 3.4.8 update until this is solved.

**Comment by brianteeman** [comment]
Where did you download the file from?
I can NOT confirm that there is any problem in the update files

On 4 January 2016 at 12:00, Eric Plugge <notifications@github.com> wrote:

> This morning I've updated my Joomla site from 344 to 348 thru
> Administrator A few moments later the website' files have the extension
> ENCRYPTED and indeed are not readilble anymore I also found a
> README_FOR_DECRYPTtxt file with payment instructions
>
> The website is hosted at wwwmijndomeinnl, and well known provider in the
> Netherlands
>
> Of course I don't know how this was possible But it looks like the
> contamination came with the update So, I guess you should warn others too,
> and maybe prevent the 348 update until this is solved
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-websites/issues/202>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by ericplugge** [comment]
Hi Brian,

Just followed the panel to install the update, cleard the cache, opted for writing files directly. So, no download of files whatsoever.

Kind regards, Eric

**Comment by ericplugge** [comment]
I also did 2 other website and they seem not to be affected.

**Comment by brianteeman** [comment]
The Update component takes its files from here
https://github.com/joomla/joomla-cms/releases/tag/3.4.8
As you can see they have not been updated for 11 days and you can download
them manually and confirm that they are not "infected" which you yourself
have confirmed by updating other sites with no problem

On 4 January 2016 at 12:07, Eric Plugge <notifications@github.com> wrote:

> I also did 2 other website and they seem not to be affected.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-websites/issues/202#issuecomment-168660098>
> .
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by ericplugge** [comment]
Hi Brian,

Since I had no connection to the website other than thru the Administrator's Control Panel for the update to 3.4.8 it seems to me that the vulnerability took place somewhere between login and the installation of the new version. Because it was possible that existing files are renamed does suggest that something is running on the webserver to make this possible.

I'm sure others will encounter this problem as well. I have informed my provider and will let you know what they have to tell.

If you have any other suggestions or theories, please let me know!

Kind regards, Eric

**Comment by brianteeman** [comment]
Yes it sounds likes your webserver has been hacked
No there is nothing worng with the joomla update files

On 4 January 2016 at 12:21, Eric Plugge <notifications@github.com> wrote:

> Hi Brian,
>
> Since I had no connection to the website other than thru the
> Administrator's Control Panel for the update to 3.4.8 it seems to me that
> the vulnerability took place somewhere between login and the installation
> of the new version. Because it was possible that existing files are renamed
> does suggest that something is running on the webserver to make this
> possible.
>
> I'm sure others will encounter this problem as well. I have informed my
> provider and will let you know what they have to tell.
>
> If you have any other suggestions or theories, please let me know!
>
> Kind regards, Eric
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-websites/issues/202#issuecomment-168664992>
> .
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by mikeveeckmans** [comment]
I agree with @brianteeman  ... Have done +25 upgrades to 3.4.8 not occurred a single similar issue.

**Comment by ericplugge** [comment]
I have a full copy of the website files from before the update. Are you interested to have a look and see if you could detect anything?

**Comment by ericplugge** [comment]
I found the ransomware. The name of the file is pagenews.php and was placed in the root of the website. Windows Defender recognises it as Backdoor:PHP/Shell.G, severe.

**Comment by ericplugge** [comment]
Okay, last reply. The update 3.4.8 does not contain the files that I have found on my website that contained the ransomware. That is good news! Still I'm a little nervous how it was possible to add these files to the website.

But perhaps we never know. I have requested my provider to restore an older backup. This result looks fine, and hopefully they will set the website back online soon.

Kind regards, Eric
